test/statsCases/module-deduplication/webpack.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
 	},
 	stats: {
 		hash: false,
-		timing: false,
+		timings: true,
 		chunks: true,
```

Comment:
Could you change these both to false and adapt the expected output? This was how it was intended.

Reply:
Sorry about that – missed your message. Updated in #5835.